validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -106,7 +118,13 @@ def __init__(self,
         Returns:
             None
         """
-        self._block_cache = block_cache
+
+        self._block_manager = block_manager
+        self._block_store = block_store
+        self._chain_head_fn = None
+        self._block_validity_fn = None
+        self._block_validity_cache = _Cache(100)
```

Comment:
This 100 is meaningless without implementing the capacity limit, so should be dropped until that is implemented.

Reply:
Thanks. As discussed I will modify the ChainController/BlockValidator/BlockScheduler interaction so that this cache is not necessary.